app/blog/article/[slug]/components/BlogPopup.js

```diff
@@ -1,55 +1,33 @@
 'use client';
 import { useEffect, useState } from 'react';
-import HubSpotForm from '@shared/utils/HubSpotForm';
 import SignupForm from '@shared/inputs/SignupForm';
-import ModalCorner from '@shared/utils/ModalCorner';
 import Image from 'next/image';
 import { setCookie, getCookie } from 'helpers/cookieHelper.js';
 import Modal from '@shared/utils/Modal';
 
+const POPUP_DELAY = 30000;
 export default function BlogPopup() {
   const [showModal, setShowModal] = useState(false);
-  const [testVariant, setTestVariant] = useState(false);
 
   const handleOpenModal = () => {
     setShowModal(true);
   };
 
   useEffect(() => {
     const cookie = getCookie('blogPopup');
-    const timerTest = setTimeout(() => {
-      if (window.gpBlogPopupTest) {
-        //this is setup as a js variable in blog test on VWO
-        setTestVariant(true);
-      }
-    }, 500);
     if (!cookie || cookie !== 'closed') {
-      const timer = setTimeout(handleOpenModal, 30000);
+      const timer = setTimeout(handleOpenModal, POPUP_DELAY);
       return () => {
         clearTimeout(timer);
-        clearTimeout(timerTest);
       };
     }
   }, []);
 
-  useEffect(() => {
-    if (window.gpBlogPopupTest) {
-      setTestVariant(true);
-    }
-  }, []);
-
-  const WrapperElement = (props) => {
-    if (testVariant) {
-      return (
-        <Modal {...props}>
-          <div className=" w-[90vw] max-w-[420px] mx-auto">
-            {props.children}
-          </div>
-        </Modal>
-      );
-    }
-    return <ModalCorner {...props}>{props.children}</ModalCorner>;
-  };
+  const WrapperElement = (props) => <Modal {...props}>
+    <div className=" w-[90vw] max-w-[420px] mx-auto">
+      {props.children}
+    </div>
+  </Modal>
```

Comment:
We needed the wrapper element for the test logic. Now that you removed it, we can just move this code to the return statement and remove the WrapperElement